packages/gatsby-theme-aio/src/components/GetCredential/CredentialForm.js

```diff
@@ -130,33 +130,23 @@ const CredentialForm = ({
   }, []);
 
   useEffect(() => {
-    if (isError) {
-      const updateForm = { ...formData };
-      for (const key in updateForm) {
-        updateForm[key] = '';
+    const isValidCredentialName = credentialNameRegex.test(formData.CredentialName) && formData['CredentialName']?.length >= 6;

```

Comment:
Isn't the length check covered by the regex?

Reply:
Okay I'll change